packages/sign-plugin/src/utils/pluginValidation.test.ts

```diff
@@ -0,0 +1,15 @@
+import { getPluginJson, validatePluginJson } from './pluginValidation';
+
+describe('pluginValidation', () => {
+  describe('plugin.json', () => {
+    test('missing plugin.json file', () => {
+      expect(() => getPluginJson(`${__dirname}/mocks/missing-plugin.json`)).toThrowError();
+    });
+  });
+
+  describe('validatePluginJson', () => {
+    test('missing plugin.json file', () => {
```

Comment:
Just a nit typo, probably not important at all. 
```suggestion
    test('missing "id" field in the plugin.json file', () => {
```
(I also know that the code is coming from the Grafana repo as it was and that these tests are not covering too much, we probably improve these in an upcoming PR(s) anyway.)